x/campaign/types/params.go

```diff
@@ -11,10 +11,12 @@ import (
 )
 
 var (
-	DefaultMinTotalSupply = sdk.NewInt(100)                   // One hundred
-	DefaultMaxTotalSupply = sdk.NewInt(1_000_000_000_000_000) // One Quadrillion
+	DefaultMinTotalSupply      = sdk.NewInt(100)                   // One hundred
+	DefaultMaxTotalSupply      = sdk.NewInt(1_000_000_000_000_000) // One Quadrillion
+	DefaultCampaignCreationFee = sdk.Coins(nil)                    // EmptyCoins
 
-	ParamStoreKeyTotalSupplyRange = []byte("totalsupplyrange")
+	ParamStoreKeyTotalSupplyRange    = []byte("totalsupplyrange")
+	ParamStoreKeyCampaignCreationFee = []byte("campaigncreationfee")
```

Comment:
```suggestion
	KeyTotalSupplyRange    = []byte("TotalSupplyRange")
	KeyCampaignCreationFee = []byte("CampaignCreationFee")
```

To be consistent with other module